docs/user-guide/zowe-secure-setup.md

```diff
@@ -0,0 +1,225 @@
+- z/OSMF High Availability mode should be configured. See [Configuring z/OSMF high availability in Sysplex](../user-guide/systemrequirements-zosmf-ha).
+
+Instance on every LPAR is started.
+
+**Configuration**
+
+The configuration for the specific instance is composed of the defaults in the main section and the overrides in the `haInstances` section.
+
+In this section, <ha-instance> represents any Zowe high availability instance ID. Every instance has internal id and a section with overrides compared to the main configuration in the beginning of the `zowe.yaml` file. Check [Zowe YAML configuration reference](../appendix/zowe-yaml-configuration/#yaml-configurations---hainstances) for details.
+
+### Caching service setup and configuration
+
+Zowe uses the Caching Service to centralize the state data persistent in high availability (HA) mode. It can be used to share information between services.
+
+If you are runnning the caching service on z/OS, there are three storage methods with their own characteristics:
+
+- [VSAM](../user-guide/configure-caching-service-ha)
+  - Known by zOS engineers.
+  - Slow.
+- [Redis](../extend/extend-apiml/api-mediation-redis#redis-configuration)
+  - Needs to run in Distributed world separately.
+  - Good for Kubernetes deployment.
+- [Infinispan (*recommended*)](../extend/extend-apiml/api-mediation-infinispan#infinispan-configuration)
+  - Part of the Caching service.
+  - Doesn’t need separate processes.
+  - Highly performant.
+
+## Observation
+
+It is essential to have observation in the whole secure operation process. The service availability is visible to everyone. It’s easy to find performance issues with southbound services.
+
+To do so, Zowe can easy integrate with the Alerting and Monitoring enterprise services such as ELK, Splunk, Grafana, etc and integrate the data as part of the Zero Trust Architecture approach with, for example SIEM (Security Incident Event Management). 
+
+### Use Audit Log to observe
+
+You can install the API Audit Log for Zowe API ML. The API `Audit Log` is an add-on to Zowe API Mediation Layer that provides unified auditing logs.
+
+**Prerequisites**
+
+- Brightside license is required.
+- Elasticsearch, Logstash, and Kibana (ELK Stack) is configured. Using the API Audit Log also enables you to utilize ELK Stack for data visualizations.
+
+For detailed information about the API Audit Log, see how to [Enable the API Audit Log](https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/4-0/installing/install-brightside-extensions-and-z-os-components/enable-the-api-audit-log.html).
+
+### Features
+
+The API `Audit Log` can help you observe and analyze the data located on the mainframe, including suspicious activities and the state of the services.
+
+## Distributed Identify Federation
```

Comment:
This topic could be expanded to provide extended information.

Reply:
Hi Pavlin, thank you for your detailed comments and suggestions. Just wondering if you want this section to be another new page or just add more information here. Would you please leave some input or reference about the extended information you want here?

Reply:
> This topic could be expanded to provide extended information.

Hi Pavlin, thank you for your detailed comments and suggestions. Just wondering if you want this section to be another new page or just add more information here. Would you please leave some input or reference about the extended information you want here?

Reply:
I think the topic deserves separate page (which don't have yet) and then could be introduced here with link to the detailed explanation.

Reply:
> I think the topic deserves separate page (which don't have yet) and then could be introduced here with link to the detailed explanation.

Will do. Before that, I'm wondering if the page will just cover the information like this draft, or you have further input to this page.